doc/api/next_api_changes/removals/22107-OG.rst

```diff
@@ -0,0 +1,44 @@
+Removal of deprecated ``mathtext`` APIs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following :mod:`.mathtext` APIs have been removed:
```

Comment:
```suggestion
The following `matplotlib.mathtext` APIs have been removed:
```

We use 'py:obj' as sphinx default role, so we can leave out `:mod:` and similar qualifiers, which improves readability of the plain texts.

Reply:
OK! I used https://github.com/matplotlib/matplotlib/blob/main/doc/api/next_api_changes/removals/22081-AL.rst as a reference, but I see your point.

Reply:
It's more important in docstrings than in rst-files. But I tend to use the same everywhere.